devel/smake/Portfile

```diff
@@ -1,41 +1,56 @@
-PortSystem 1.0
-PortGroup  compiler_blacklist_versions 1.0
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
+
+name			        smake
```

Comment:
I suggest to add a mode line https://guide.macports.org/#development.creating-portfile and reformat this file, because via github UI it looks quite messy

Reply:
Well do.  I did not realize that the original Portfile for `smake` was missing that line.  Thanks for pointing that out.

Reply:
Done and forced pushed.

Reply:
Now I see warnings -- tabs in the Portfile.  I will fix that.